tutorials/quickstarts/sklearn-mnist-batch/utils.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
remove license - this file can probably just be a cell in the relevant notebook(s)

Reply:
License removed.